hugophotoswipe/photo.py

```diff
@@ -78,14 +82,13 @@ def original_image(self):
     def _load_original_image(self):
         img = Image.open(self.original_path)
         # if there is no exif data, simply return the image
-        exif = img._getexif()
+        self._load_exif(img)
+        exif = self._exif
         if exif is None:
             return img
 
         # get the orientation tag code from the ExifTags dict
-        orientation = next(
-            (k for k, v in ExifTags.TAGS.items() if v == "Orientation"), None
-        )
+        orientation = exif.get('Orientation')
         if orientation is None:
             print("Couldn't find orientation tag in ExifTags.TAGS")
```

Comment:
This warning is triggered for me now when running the test cases, could this be because of your changes?

Reply:
This was an old warning that sneaked back in. Removed.